### PR TITLE
Hermes charts

### PR DESCRIPTION
**Pull request by pedroborgeszup**

### CODE REVIEW

File: install/helm-chart/templates/charlescd-hermes-secret.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{{- if .Values.compass.enabled }}
```
Comment (barbararochazup) [inline install/helm-chart/templates/charlescd-hermes-secret.yaml:1]: Values.compass.enabled tá correto?

File: install/helm-chart/charts/rabbitmq/values.schema.json [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        }
+      }
+    },
+    "metrics": {
+      "type": "object",
+      "form": true,
+      "title": "Prometheus metrics details",
+      "properties": {
+        "enabled": {
+          "type": "boolean",
```
Comment (barbararochazup) [inline install/helm-chart/charts/rabbitmq/values.schema.json:77]: essa parte do prometheus foi adicionada? ou ja existia?

**Review by barbararochazup — COMMENTED** [review]
algumas duvidas, somente

File: install/helm-chart/templates/charlescd-hermes-secret.yaml (hunk 1 above)
Comment (pedroborgeszup) [reply install/helm-chart/templates/charlescd-hermes-secret.yaml:1]: Ctrl + C and Ctrl + V problems, sorry! hahah

File: install/helm-chart/charts/rabbitmq/values.schema.json (hunk 2 above)
Comment (pedroborgeszup) [reply install/helm-chart/charts/rabbitmq/values.schema.json:77]: These files from RabbitMQ come like this by default after unzipping the .tgz.

Comment (pedroborgeszup) [reply install/helm-chart/charts/rabbitmq/values.schema.json]: This files wasn't necessary.

File: install/helm-chart/requirements.yaml [hunk 3]
```diff
@@ -17,4 +17,6 @@ dependencies:
 - name: ingress-nginx
   version: "3.15.2"
   repository: "https://kubernetes.github.io/ingress-nginx"
-  condition: nginx_ingress_controller.enabled
\ No newline at end of file
+  condition: nginx_ingress_controller.enabled
+- name: rabbitmq
+  version: "3.8.11"
```
Comment (lucassaleszup) [inline install/helm-chart/requirements.yaml:22]: add a condition rabbitmq.enabled

Comment (lucassaleszup) [reply install/helm-chart/requirements.yaml:22]: and the repository

**Review by lucassaleszup — CHANGES_REQUESTED** [review]

**Review by lucassaleszup — APPROVED** [review]

File: install/helm-chart/requirements.yaml (hunk 3 above)
Comment (pedroborgeszup) [reply install/helm-chart/requirements.yaml]: Done

**Review by barbararochazup — APPROVED** [review]

**Review by celiofilhozup — APPROVED** [review]

**Pull request merged**